android-components/components/ui/icons/src/main/res/drawable/mozac_ic_private_mode_24.xml

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We shouldn't remove this since this is still part of the icons design library.

Reply:
restored this icon